### DISCUSSION TITLE
Questions about docker images linked to deepxde documentation

### DISCUSSION THREAD

**Discussion by ParkBeomChul** [lululxvi/deepxde#1192] [Q&A]
Questions about docker images linked to deepxde documentation

Hello, thank you for your hard work.

I installed deepxde using the docker image (pescapil/deepxde:latest) linked to deepxde documentation 1.8.1. However, the pytorch version does not fit and therefore does not work. Could you modify the docker image?
(Required version 1.9.0 /  Installed version 1.8.1)

Thank you.

**Comment by pescap** [comment]
I'll work on that!

**Reply by pescap** [reply]
Solved with #1202

**Reply by pescap** [reply]
Update the docker image:

```
docker pull pescapil/deepxde:latest
```

and run an instance, with e.g.:
```
nvidia-docker run -v $(pwd):/root/shared -w "/root/shared" -p 8888:8888 pescapil/deepxde:latest
```
